credential/manifest/model.go

```diff
@@ -141,4 +141,80 @@ func (cf *CredentialResponse) IsValid() error {
 	return util.NewValidator().Struct(cf)
 }
 
+// IsValidPair validates the rules on how a credential manifest [cm] and credential application [ca] relate to each other https://identity.foundation/credential-manifest/#credential-application
+func IsValidPair(cm CredentialManifest, ca CredentialApplication) error {
```

Comment:
consider making this a receiver such as

`func (cm CredentialManifest) IsValidApplication(ca CredentialApplication)`

this may be confusing. if it is, maybe make the method name more explicit - something like `IsValidCredentialApplicationForManifest()`...

Reply:
Yup I thought the name was off, I'll name it to IsValidCredentialApplicationForManifest